### PR TITLE
Improve performance by placeholdering all blocks using our cookie con…

### PR DESCRIPTION
**Pull request by nikunjkotecha**
…text.

### CODE REVIEW

File: src/AcmServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+namespace Drupal\acm;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceModifierInterface;
+use Drupal\Core\DependencyInjection\ServiceProviderBase;
+
+/**
+ * Class AcmServiceProvider.
+ */
+class AcmServiceProvider extends ServiceProviderBase implements ServiceModifierInterface {
```
Comment (sdelbosc) [inline src/AcmServiceProvider.php:12]: I am confused about this and #6504

Comment (nikunjkotecha) [reply src/AcmServiceProvider.php:12]: class name needs to match module name

Comment (sdelbosc) [reply src/AcmServiceProvider.php:12]: it was more about potential duplication, we will need to discuss

Comment (nikunjkotecha) [reply src/AcmServiceProvider.php:12]: @sdelbosc we discussed, this is fine.

**Pull request merged**